### PR TITLE
[Workers] wrangler meta descriptions

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-8824

### CODE REVIEW

File: content/workers/wrangler/system-environment-variables.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
 pcx_content_type: configuration
 title: System Environment Variables
+meta:
+  description: Local environment variables that can change Wrangler's behavior.
```
Comment (penalosa) [inline content/workers/wrangler/system-environment-variables.md:5]:
```suggestion
  description: System environment variables that can change Wrangler's behavior.
```

**Review by penalosa — APPROVED** [review]

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ pcx_content_type: reference
 title: Commands
 meta:
   title: Commands - Wrangler
+  description: Create, develop and deploy your Cloudflare Workers with Wrangler commands.
```
Comment (pedrosousa) [inline content/workers/wrangler/commands.md]:
```suggestion
  description: Create, develop, and deploy your Cloudflare Workers with Wrangler commands.
```

**Review by marciocloudflare — APPROVED** [review]

File: content/workers/wrangler/system-environment-variables.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
 ---
 pcx_content_type: configuration
-title: System environment variables
+title: System Environment Variables
```
Comment (pedrosousa) [inline content/workers/wrangler/system-environment-variables.md]: Nit: Not sure if this is a Cloudflare concept that needs title case.

**Review by pedrosousa — APPROVED** [review]

File: content/workers/wrangler/system-environment-variables.md (hunk 3 above)
Comment (deadlypants1973) [inline content/workers/wrangler/system-environment-variables.md]:
```suggestion
title: System environment variables
```

**Pull request merged**